Sources/Keyboard/Keyboard.swift

```diff
@@ -10,19 +10,22 @@ public struct Keyboard<Content>: View where Content: View {
     var latching: Bool
     var noteOn: (Pitch) -> Void
     var noteOff: (Pitch) -> Void
+    var rows: Int
     var layout: KeyboardLayout
 
     public init(pitchRange: ClosedRange<Pitch> = (Pitch(60)...Pitch(72)),
                 latching: Bool = false,
                 layout: KeyboardLayout = .piano,
                 noteOn: @escaping (Pitch) -> Void = { _ in },
                 noteOff: @escaping (Pitch) -> Void = { _ in },
+                rows: Int = 6,
```

Comment:
RowCount is ignored for everything but guitar. Maybe either make the others types react to `rowCount`or call it `guitarStringCount` (less preferable).